.changeset/weak-crews-judge.md

```diff
@@ -0,0 +1,80 @@
+---
+'@razorpay/blade': major
+---
+
+feat(Typography): streamline typography scale
+
+> **Warning**
+>
+> Breaking Change!
+> This is a breaking change for typography components and lineHeight scale
+
+### Component Changes:
+
+- Title:
+  - Added `xlarge` size
+- Text:
+  - Added `large` size
+- Link:
+  - Added `large` size
+- Code:
+  - Added `weight=bold`
+
+### Breaking Changes:
+
+- Title:
+  - Replace all `large` size variants to `xlarge`
+
```

Comment:
can you add a VScode global find and replace as a snippet here?

Reply:
also add there are just 5 instances across razorpay with `size="large"` from our analytics data so this doesn't needs a codemod

Reply:
Yes, will also notify the respective teams about this 5 changes.

Reply:
> can you add a VScode global find and replace as a snippet here?

Don't think there will be any easy way to do search and replace with vscode, they will just have to do the old fashioned  way. 

Since: 
- The quotes can change like single quotes or double quotes
- There can be line-breaks and other props used in the title which will break the search and replace query